apiserver/hack/kind-cluster-config.yaml

```diff
@@ -31,3 +20,7 @@ nodes:
   image: kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb
 - role: worker
   image: kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb
+- role: worker
+  image: kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb
+- role: worker
+  image: kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb
```

Comment:
I could not start all 4 workers on my VM. One of the kube-proxy pod says `1 run.go:74] "command failed" err="failed complete: too many open files"` but running 3 workers was fine.

Reply:
what your VM spec ? Number of CPU(s) and memory ? Also, if you type `docker info` what are the values for these fields (this what I have set on my M1 mac):

```text
 OSType: linux
 Architecture: aarch64
 CPUs: 6
 Total Memory: 15.6GiB
```

Reply:
@tedhchang and @blublinsky can you add your docker info output here ?

Reply:
sure @z103cb 
```
docker info
Client:
 Context:    default
 Debug Mode: false

Server:
 Containers: 6
  Running: 4
  Paused: 0
  Stopped: 2
 Images: 7
 Server Version: 20.10.21
 Storage Driver: overlay2
  Backing Filesystem: xfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version:
 runc version:
 init version:
 Security Options:
  apparmor
  seccomp
   Profile: default
  cgroupns
 Kernel Version: 5.15.0-84-generic
 Operating System: Ubuntu 22.04 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 16
 Total Memory: 31.34GiB
 Name: xx
 ID: 5TPO:ZCMS:RKUY:WJR6:4TIU:KER2:TCW3:YZNN:O2DD:RYA5:ETTQ:DOQ7
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Username: x
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```